README.md

```diff
@@ -126,3 +126,72 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## pre deployment to fly.io
+
+
+- Setting Up environmental variables
+
+  - Add `python-dotenv>=0.21.0` to requirements.txt
+  - Run `pip install -r requirements.txt`
+
+  -  create new .env file
+      add the following informations in it
+          SECRET_KEY
+          DATABASE_URL
```

Comment:
And how do you get this?

Reply:
I generated the scring with separate python string and put it inside .env file and access it as environment variable with os.getenv('SECRET_KEY") from .env by using library i added before

Reply:
Again. The README should be a simple list of bash commands that should be copy pasted. Not abstract instructions